tests/languages/lisp/string_feature.test

```diff
@@ -0,0 +1,29 @@
+(
+""
+"foo
+bar"
+"\"foo\""
+"foo\tbar"
+"foo\tbar"

```

Comment:
Looks like there's a duplicate here. I don't think there's a need to test it twice.